src/System.Reflection.Context/pkg/System.Reflection.Context.pkgproj

```diff
@@ -2,7 +2,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.Reflection.Context.csproj">
-      <SupportedFramework>uap10.0.16299;netcoreapp2.0;net45;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
Technically I think you didn't need to remove `$(AllXamarinFrameworks)` here.  It's still working correctly because we're harvesting the supported version from the previous package and the assembly version of ref didn't change.

Reply:
At least for one of those two projects it was failing in the allconfigurations leg. Can't remember exactly which one it was anymore.